macros/schema.yml

```diff
@@ -3,12 +3,44 @@ version: 2
 macros:
   - name: channel_classification
     description: A macro used to perform channel classifications
-
   - name: channel_spend
     description: A macro for the user to overwrite it with a sql script to extract total ad spend by channel
-
   - name: conversion_clause
     description: A macro to specify how to filter on conversion events
-
   - name: conversion_value
     description: A macro that specifies either a single column or a calculated value that represents the value associated with the conversion
+  - name: get_lookback_date_limits
+    description: A macro returning the upper or lower boundary to limit what is processed by sessions_by_customer_id
+    arguments:
+      - name: limit_type
+        type: string
+        description: Can be either 'min' or 'max' depending on if the upper or lower boundary date needs to be returned
+  - name: create_udfs
+    description: Creates user defined functions for adapters apart from Databricks. It is executed as part of an on-start hook.
+  - name: path_transformation
+    description: Macro to execute the indvidual path_transformation specified as a parameter
+    arguments:
+      - name: transformation_type
+        type: string
+        description: A type of transformation that needs to be executed E.g. 'unique_path'
```

Comment:
Should we list the specific options (for databricks) here? (as well as it's already in the main docs)

Reply:
Also maybe we could put a link in here or the macro description to https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-fractribution-data-model/#path-transform-options just to save duplicating words, I find having the examples on hand infinitely valuable (side note that page needs the two `remove` options added)